vim/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+
+
+if [ "$1" = "vim" ]; then
+   adduser -h /home/$NAMESPACE -s /bin/bash -G root -D -H $NAMESPACE
+   mkdir -p /home/$NAMESPACE
```

Comment:
Using built-in Kubernetes envvars makes our images Kubernetes-dependent, meaning that other Docker-based projects may not be able to leverage them as easily.

This should be avoided if at all possible.